Mathlib/Algebra/Lie/Semisimple/Basic.lean

```diff
@@ -304,18 +304,13 @@ instance (priority := 100) IsSimple.instIsSemisimple [IsSimple R L] :
 /-- An abelian Lie algebra with trivial radical is trivial. -/
 theorem subsingleton_of_hasTrivialRadical_lie_abelian [HasTrivialRadical R L] [h : IsLieAbelian L] :
     Subsingleton L := by
-  rw [isLieAbelian_iff_center_eq_top R L, HasTrivialRadical.center_eq_bot] at h
+  rw [isLieAbelian_iff_center_eq_top R L, center_eq_bot] at h
   exact (LieSubmodule.subsingleton_iff R L L).mp (subsingleton_of_bot_eq_top h)
 
 theorem abelian_radical_of_hasTrivialRadical [HasTrivialRadical R L] :
     IsLieAbelian (radical R L) := by
   rw [HasTrivialRadical.radical_eq_bot]; exact LieIdeal.isLieAbelian_of_trivial ..
 
-/-- The two properties shown to be equivalent here are possible definitions for a Lie algebra
-to be reductive.
-
-Note that there is absolutely [no agreement](https://mathoverflow.net/questions/284713/) on what
-the label 'reductive' should mean when the coefficients are not a field of characteristic zero. -/
```

Comment:
Why are you removing this comment now?
Aha, I see, you moved it to the docstring of `HasCentralRadical`.

But maybe this docstring can point there?
```suggestion
/-- The two properties shown to be equivalent here are possible definitions for a Lie algebra
to be reductive.
See also `LieAlgebra.HasCentralRadical`. -/
```

Reply:
Sadly that comment was wrong: it only assumes that the radical is abelian which is weaker than the radical being central!